R/apply_formats.R

```diff
@@ -0,0 +1,61 @@
+#' Apply Format Strings outside of a Tplyr table
+#'
+#' The `f_str` object in Tplyr is used to drive formatting of the outputs
+#' strings within a Tplyr table. This function allows a user to use the same
+#' interface to apply formatted string on any data frame within a
+#' `dplyr::mutate()` context.
+#'
+#' Note that auto-precision is not currently supported within `apply_formats()`
+#'
+#' @param format_string The desired display format. X's indicate digits. On the
+#'   left, the number of x's indicates the integer length. On the right, the
+#'   number of x's controls decimal precision and rounding. Variables are
+#'   inferred by any separation of the 'x' values other than a decimal.
+#' @param ... The variables to be formatted using the format specified in
+#'   \code{format_string}. These must be numeric variables.
+#' @param empty The string to display when the numeric data is not available.
+#'   For desc layers, an unnamed character vector will populate within the
+#'   provided format string, set to the same width as the fitted numbers. Use a
+#'   single element character vector, with the element named '.overall' to
+#'   instead replace the whole string.
```

Comment:
I think we can remove this part of the argument description as its no longer relevant to count layers

Reply:
@elimillera that's not how the empty currently works. it's not a named parameter, but either one token to fill each space or .overall to fill all. Room for improvement on the functionality but not yet implemented.